package.json

```diff
@@ -42,6 +43,8 @@
     "browserify": "^12.0.1",
     "eslint": "^3.19.0",
     "jasmine": "^2.5.3",
+    "jsesc": "^2.5.1",
+    "unicode-9.0.0": "^0.7.4",
```

Comment:
I’ve used unicode-9.0.0 (and not unicode-10.0.0) here to more clearly show the actual diff. We can easily update to unicode-10.0.0 later by just swapping out the dependency (i.e. changing this line) and then running `npm run build-unicode-data` again. No more manually updating Unicode data files and Python scripts!

Reply:
Yeah, I think this is the way to go! I may also look into using a modified version of [prettier-diff](https://github.com/josephfrazier/prettier-diff) to ignore differences in the object key ordering.

Reply:
I tried to make sure the order remains the same. AFAICT the only difference is that the property names are now quoted (jsesc does this), i.e. `name: 'Foo'` becomes `'name': 'Foo'`, and that the escaping for various characters is different.